pfe: #namespace

    path: '/home/desire/catkin_ws/src/pfe_project'
    laser_to_skip: 90 # The number of laser scans to skip scan = total_scans % laser_to_skip

    # q_learning parameters
    naction: 3
    alpha: 0.6
    gamma: 0.7
    epsilon: 0.75
    epsilon_discount: 0.998
    nepisodes: 1000
    nsteps: 1000
    number_splits: 10 

    # Reward parameters
    forwards_reward: 10 # Points Given to go forwards
    turn_reward: 1 # Points Given to turn as action
    end_episode_reward: 200 # Points given when ending an episode (negative value for failing)